monai/data/decathlon_datalist.py

```diff
@@ -186,3 +187,39 @@ def check_missing_files(
                     missing_files.append(f)
 
     return missing_files
+
+
+def create_cross_validation_datalist(
```

Comment:
I don't think this utility adds enough value, partition_dataset and fold selection are already easy-to-use...

Reply:
Hi @wyli and @ericspod ,

Thanks for your review and comments.
This utility is also very useful in NVIDIA Clara project to help users automatically create train / val split for the datalist as all the Clara datalists follow Decathlon style. I think it would be nice to have this utility in MONAI directly.
What do you guys think?

Thanks.

Reply:
Ok, we can add this, I guess it's more attractive if we provide a command line interface in the future.

Reply:
Hi @wyli ,

I have updated the PR according to our online discussion.
Could you please help review it again?
Please note that the datalist of JSON file can be strings, list or numbers, but can't be `Path`, so I didn't add the logic to allow `Path` in datalist content.

Thanks.

Reply:
Thanks, I think the  filename variable can be Path.

Reply:
Yes, I already added `Path` support to `filename` in the previous commit.

Thanks.

Reply:
Sure, I didn't ask for path in the JSON, but they should be allowed in the missing file check though

Reply:
Sure, I have updated the PR with `Path` and `raise_error` flag.
Could you please help review it again?

Thanks in advance.